src/patternfly/components/DualListSelector/dual-list-selector.scss

```diff
@@ -259,6 +260,14 @@ $pf-c-dual-list-selector__item--MaxNesting: 10;
   text-align: left;
   cursor: pointer;
   border: 0;
+
+  &.pf-m-disabled {
+    .pf-c-dual-list-selector__item-text {
+      --pf-c-dual-list-selector__item-text--Color: var(--pf-c-dual-list-selector__list-item-row--m-disabled__text--Color);
```

Comment:
You can just set this var under `&.pf-m-disabled` - should be no need to scope it to `&.pf-m-disabled .pf-c-dual-list-selector__item-text`